django/contrib/admin/static/admin/js/calendar.js

```diff
@@ -8,8 +8,25 @@ depends on core.js for utility functions like removeChildren or quickElement
     'use strict';
     // CalendarNamespace -- Provides a collection of HTML calendar-related helper functions
     var CalendarNamespace = {
-        monthsOfYear: gettext('January February March April May June July August September October November December').split(' '),
-        daysOfWeek: gettext('S M T W T F S').split(' '),
+        monthsOfYear = [gettext('January'),
+                        gettext('February'),
+                        gettext('March'),
+                        gettext('April'),
+                        gettext('May'),
+                        gettext('June'),
+                        gettext('July'),
+                        gettext('August'),
+                        gettext('September'),
+                        gettext('October'),
+                        gettext('November'),
+                        gettext('December')],
```

Comment:
You need to use `:` instead of `=`.

Plus I would de-indent the array content to match the current indentation.

e.g.

```javascript
 monthsOfYear: [
    gettext('January'),
    ....
    gettext('December')
]
```

Instead of

```javascript
 monthsOfYear: [gettext('January'),
                          ...
                          gettext('December')]
```